packages/transaction-controller/package.json

```diff
@@ -55,6 +56,7 @@
     "@babel/runtime": "^7.23.9",
     "@metamask/auto-changelog": "^3.4.4",
     "@metamask/ethjs-provider-http": "^0.3.0",
+    "@types/bn.js": "^5.1.5",
```

Comment:
This one is okay, I think — I see this showing up in `packages/transaction-controller/dist/helpers/EtherscanRemoteTransactionSource.js` but not the corresponding `.d.ts` file.